.circleci/config.yml

```diff
@@ -298,10 +298,10 @@ jobs:
         env:
           TOX_SKIP_DIST: True
           CASS_DRIVER_NO_EXTENSIONS: 1
-      - image: cassandra:3.11
+      - image: spotify/cassandra:latest
         env:
-          - MAX_HEAP_SIZE=1024M
-          - HEAP_NEWSIZE=400M
+          - MAX_HEAP_SIZE=512M
```

Comment:
Can you comment on these values?

Reply:
One of the issues with the flakiness was running out of memory, if we can get by with reducing the allowed memory size for cassandra, then might as well.


more in the "gut instinct" realm than actual science... it works ™️

Reply:
https://docs.oracle.com/javase/8/docs/technotes/guides/vm/gc-ergonomics.html

```
initial heap size

Larger of 1/64th of the machine's physical memory on the machine or some reasonable minimum. Before Java SE 5.0, the default initial heap size was a reasonable minimum, which varies by platform. You can override this default using the -Xms command-line option.

maximum heap size

Smaller of 1/4th of the physical memory or 1GB. Before Java SE 5.0, the default maximum heap size was 64MB. You can override this default using the -Xmx command-line option.
```